### PR TITLE
add container-technologies-market-analysis

### PR DESCRIPTION
**Pull request by rose-coste**
This is a PR for editorial review.
It continues and replaces PR #165, which collected technical review feedback in the private repo.
It responds to issue #138

### CODE REVIEW

**Comment by stephamon** [comment]
Hi @rose-coste I'm attaching a Word doc with edits. Let me know if you have any questions or comments!

[container-technologies-market-analysis_SFedits.docx](https://github.com/rackerlabs/docs-container-service/files/18698/container-technologies-market-analysis_SFedits.docx)

**Comment by rose-coste** [comment]
Hi, @stephamon, I've incorporated your feedback. Please let me know what you think.

File: _best-practices/container-technologies-market-analysis.md [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+
+In short, the virtualization we’ve all come to know and use today is
+possible because of the development and rise of the hypervisor. If virtualization as enabled by hypervisors has been so ubiquitous and successful, why has the focus changed to containers and Docker? James Bottomley, leading Linux kernel
+developer and CTO of server virtualization for Parallels, explained that VM hypervisors, such as Hyper-V, KVM, and Xen, are all "based on emulating virtual hardware. That means
+they’re fat in terms of system requirements" [(1)](#resources).
+
+Containers, however, use shared operating systems. That means they are
+much more efficient than hypervisors in terms of system resources. Instead
+of virtualizing hardware, containers rest on top of a single Linux
+instance on the host. According to Bottomley, his means that you can “leave behind
```
Comment (stephamon) [inline _best-practices/container-technologies-market-analysis.md]: Typo: According to Bottomly, this means...

File: _best-practices/container-technologies-market-analysis.md [hunk 2]
```diff
@@ -0,0 +1,195 @@
...
+the useless 99.9% VM junk, leaving you with a small, neat capsule
+containing your application” [(1)](#resources).
+
+Therefore, with a perfectly tuned container
+system, you can have as many as four-to-six times as many
```
Comment (stephamon) [inline _best-practices/container-technologies-market-analysis.md]: "as many as four-to-six times as many" is a little repetitive. Maybe "you can have as many as four-to-six times the number of server application instances..."

**Comment by stephamon** [comment]
@rose-coste I've noted two things, but otherwise it looks good! :+1:

**Pull request merged**